homeassistant/components/unifiprotect/manifest.json

```diff
@@ -40,7 +40,11 @@
   "integration_type": "hub",
   "iot_class": "local_push",
   "loggers": ["uiprotect", "unifi_discovery"],
-  "requirements": ["uiprotect==5.2.2", "unifi-discovery==1.2.0"],
+  "requirements": [
+    "uiprotect==5.2.2",
+    "unifi-discovery==1.2.0",
+    "pydantic==1.10.17"
```

Comment:
Why is this? This is an implementation detail in the client library. It leaks internals, which should be fixed.